docs/datatypes/chain.md

```diff
@@ -1,60 +1,70 @@
 # Chain
 
-`Chain` is a data structure that allows constant time prepending and appending.
-This makes it especially efficient when used as a `Monoid`, e.g. with `Validated` or `Writer`.
-As such it aims to be used where `List` and `Vector` incur a performance penalty.
+@:api(cats.data.Chain) is a data structure that allows constant time prepending, appending and concatenation.
```

Comment:
you may want to mention that actually `Chain` is simply an unbalanced binary tree with some utility methods on it for using it as a sequence, including Traverse, Monad and Alternative typeclass implementations.

Reply:
I've now added a mention of that under How it works. I don't think the opening paragraph is the right place to mention implementation details. I've added a bit more description to the opening as well though.